src/Reflection/DocBlock/Tag/PropertyTag.php

```diff
@@ -28,15 +25,11 @@ public function getName()
         return 'property';
     }
 
-    /**
-     * Initializer
-     *
-     * @param  string $tagDocblockLine
-     */
-    public function initialize($tagDocblockLine)
+    /** {@inheritDoc} */
```

Comment:
Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.

Reply:
dd68158de612598d53a7f3dcc96c27c5b3816dbf